ckcp/openshift_dev_setup.sh

```diff
@@ -301,14 +302,24 @@ install_openshfit_pipeline() {
   echo -n "  - openshift-pipeline application: "
   kubectl apply -f "$GITOPS_DIR/$APP/$APP.yaml" --wait >/dev/null 2>&1
   argocd app wait pipelines-service tektoncd --timeout=90 >/dev/null 2>&1
+  echo "OK"
+}
 
+postcheck() {
   # Wait until CRDs are synced to KCP
-  while [ "$(KUBECONFIG="$KUBECONFIG_KCP" kubectl api-resources | grep -c tekton.dev)" != 8 ]
+  local length=$(echo $TEKTON_CRDS | tr ',' '\n' | wc -l)
+  local resources_regexp=$(echo $TEKTON_CRDS | tr ',' '|')
+  local  wait_period=0
+  while [[ "$(KUBECONFIG="$KUBECONFIG_KCP" kubectl api-resources -o name | grep -E "$resources_regexp" | wc -l )" != $length ]]
   do
+    wait_period=$(($wait_period+10))
```

Comment:
```suggestion
    wait_period=$((wait_period+10))
```
IMO arithmetic calculation is better without `$`. See https://www.shellcheck.net/wiki/SC2004

Reply:
@avinal thank you for sharing the information, that's good tool for checking my script before committing. I will fix